### PR TITLE
added new cpu expression

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

**Comment by bogensberger** [comment]
ack

File: docs/sql/system.txt [hunk 1]
```diff
@@ -446,6 +446,16 @@ process
 | ``process['max_open_file_descriptors']`` | The maximum number of open file descriptors    | ``Long``     |
 |                                          | Crate can use.                                 |              |
 +------------------------------------------+------------------------------------------------+--------------+
+| ``process['cpu']``                       | Information about the system CPU               | ``Object``   |
++------------------------------------------+------------------------------------------------+--------------+
+| ``process['cpu']['percent']``            | The CPU usage of the Crate JVM process given   | ``Short``    |
+|                                          | in percent.                                    |              |
++------------------------------------------+------------------------------------------------+--------------+
+| ``process['cpu']['user']``               | The process CPU user time in milliseconds      | ``Long``     |
```
Comment (chaudum) [inline docs/sql/system.txt]: end with `.`

File: docs/sql/system.txt [hunk 2]
```diff
@@ -446,6 +446,16 @@ process
...
++------------------------------------------+------------------------------------------------+--------------+
+| ``process['cpu']['system']``             | The process CPU kernel time in milliseconds    | ``Long``     |
```
Comment (chaudum) [inline docs/sql/system.txt]: same here

**Comment by chaudum** [comment]
otherwise ack

**Pull request merged**